src/emmetHelper.ts

```diff
@@ -956,11 +956,20 @@ function getFormatters(syntax: string, preferences: any) {
 
 /**
  * Updates customizations from snippets.json and syntaxProfiles.json files in the directory configured in emmet.extensionsPath setting
+ * TODO: The input type of this setting has been changed on the vscode side. To support users with old version, the old input types are still accripted by this function.
```

Comment:
Actually, considering how the library needs to be bumped up to be used in vscode, and that the current main branch of vscode already changes the parameter to [], do we still need to keep this?

Reply:
Hm, I am thinking about for those users not update to the newest version, they might still have old value types such like `null` or `string`, so if we just removed the old types from the function. It will crash raise an error for those users.

Reply:
The main branch of vscode already has the migration function. Therefore, after we publish a new package for vscode-emmet-helper, we can directly use this package in vscode.
Also, only users of the new package will need to change their extensionsPath type.

Reply:
For reference: discussed offline, and we're keeping this part of the code so that users can set their extensionsPath like `["path1", "path2", ""]`. With this config, they get to explicitly turn off snippets if neither of the paths resolve (e.g. if they're on an environment where they don't want or care about having custom snippets).